kehan-yang.yaml

```diff
@@ -0,0 +1,26 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Kehan Yang
+  avatar: https://avatars.githubusercontent.com/u/32207757?v=4
+  role: Computational Geoscientist
+  organizations:
+    - name: M3 Works LLC
+  bio: >
+    Kehan is a computational geoscientist at M3 Works LLC, specilizing in operational snow modeling in the Western U.S.
+    She has a backaground in snow hydrology, remote sensing, and machine learning. 
+    She is passionate about investigating methods that intergate satellite data, physically-based models, and machine learning tools to 
+    enhance our understanding of mountain snowpack and water avaialbility.
+     
+  expertise:
```

Comment:
@jomey I'm not sure why this isn't rendering in the preview. Maybe the script is sensitive to newlines?